scripts/mark_eats/tuner.lua

```diff
@@ -0,0 +1,141 @@
+-- Tuner
+--
+-- Responds to audio input.
+--
+-- v1.0.0 Mark Eats
+--
+
+local MusicUtil = require "mark_eats/musicutil"
+local MEFormatters = require "mark_eats/formatters"
+
+engine.name = nil
+
+local SCREEN_FRAMERATE = 15
+local screen_dirty = true
+
+local current_freq = -1
+local last_freq = -1
+
+
+-- Encoder input
+function enc(n, delta)
+  
+  if n == 2 then
+          
+  elseif n == 3 then
+    
+  end
+end
+
+-- Key input
+function key(n, z)
+  if z == 1 then
```

Comment:
same here.  is the `key` function a placeholder for future stuff?

Reply:
Yeah I just find them useful for testing and updates but can remove if need be

Reply:
not at all.  if it's not an oversight, all good!